lib/validation.tsx

```diff
@@ -310,3 +310,72 @@ export const isValidGovEmail = (email: string, domains: string[]): boolean => {
   //Check the email's domain against the list of domains
   return domains.includes(emailDomain.toString());
 };
+
+/**
+ * This function checks if a given password is a valid email based on:
+ * 1. The policy found here found here: https://github.com/cds-snc/forms-terraform/blob/develop/aws/cognito/user_pool.tf#L5-L11
+ * 2. Additionally a max length of 50 characters
+ * And it returns true if the password is a valid otherwise false.
+ * @param password A valid password
+ * @returns {boolean} The validation result
+ */
+export const isValidPassword = (password: string): boolean => {
```

Comment:
This function is not actually used, the below ones are. I left it in encase it would be useful in a future but this is kind of a code smell. I should probably remove it - what do you think?

Reply:
Would remove it if it's not used. If you wanted to keep the regex as a reference you could add it as a comment before the functions below